dmriprep/workflows/dwi/registration.py

```diff
@@ -0,0 +1,268 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Registration workflows
+++++++++++++++++++++++
+
+.. autofunction:: init_bbreg_wf
+
+"""
+from ... import config
+
+import os
+import os.path as op
+
+import pkg_resources as pkgr
+
+from nipype.pipeline import engine as pe
+from nipype.interfaces import utility as niu, fsl, c3
+
+from ...interfaces import DerivativesDataSink
+
+DEFAULT_MEMORY_MIN_GB = config.DEFAULT_MEMORY_MIN_GB
+LOGGER = config.loggers.workflow
+
+def init_bbreg_wf(use_bbr, bold2t1w_dof, bold2t1w_init, omp_nthreads, name='bbreg_wf'):
```

Comment:
I'm a bit unclear as to what changes w.r.t. *fMRIPrep* in this implementation

Reply:
No changes from the fmriprep version here, we can just import it once it's in smriprep/niworkflows. I put it in here to get it working and I couldn't import it from fmriprep.